announcements/2021/week15.md

```diff
@@ -0,0 +1,16 @@
+# :zap: This week in provider-independent announcements (April 16, 2021) :zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, workload clusters, UI, and documentation, please check out [changes and releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## General
+
+We made a mistake, and we’re here to apologize. In our user interfaces, we previously indicated that Kubernetes v1.18 would run out of support (reach end of life, EOL) by April 8. This was a mistake. The **EOL date for v1.18 is April 14** (two days ago). Sorry for the confusion this may have caused!
+
+Our KVM team is working hard to provide a release with Kubernetes v1.19 soon.
```

Comment:
@cornelius-keller Addressing expectations here for team Rocket, as talked about on Slack.

Reply:
Looks good to me, thx